gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/internal/configureExperimental.kt

```diff
@@ -29,4 +29,5 @@ internal fun Project.configureExperimental(
             target.configureExperimentalWebApplication(experimentalExt.web.application)
         }
     }
+    project.checkExperimentalTargetsWithSkikoIsEnabled()
```

Comment:
If users don't call `configureExperimental`, they still can have experimental targets (mac os for example?).
Could this check be moved to some place where it's going to be always called? (I'm not 100% there is such a place though)

Reply:
fun configureExperimental will always call, even if compose extention not present in build.gradle.
But for better understand I can place function call to another place.

Reply:
Done, moved!

Reply:
I think now code much more understandable:
```Kotlin
configureDesktop(project, desktopExtension)
project.configureExperimental(composeExtension, experimentalExtension)
project.checkExperimentalTargetsWithSkikoIsEnabled()
```